LabelGenerator.py

```diff
@@ -117,6 +117,11 @@ def __init__(
     num_stickers_vertical=8,
 )
 
+E12 = [ 1.0, 1.2, 1.5, 1.8, 2.2, 2.7, 3.3, 3.9, 4.7, 5.6, 6.8, 8.2 ]
```

Comment:
Missing documentation on those constants. Also, maybe add `_VALUES` to the variable names, they are not very descriptive right now.

Reply:
I added documentation of the constants, along with an explanation of what the E series are. I realize most people probably don't know what E12/E24 is but I can't think of a better name, so explaining it seemed reasonable.

Added _VALUES suffix and made the difference between the two E24 values clearer.

I also moved the constants down so they live just above the methods.